src/main/java/com/example/bob_friend/service/RecruitmentService.java

```diff
@@ -51,64 +49,42 @@
 
 
     @Transactional
-    public void delete(Long recruitmentId) {
+    public void deleteRecruitment(Long recruitmentId) {
```

Comment:
이미 RecruitmentService 가 Recruitment 라는 컨텍스트를 가지고 있기 때문에 delete**Recruitment** 는 중복으로 볼수 있겠죠.

```suggestion
    public void delete(Long recruitmentId) {
```

Reply:
좀 관련 없는 질문이긴 한데, RecruitmentService의 delete를 호출하는 컨트롤러의 delete 메소드명을 보통 어떻게 짓나요? 삭제하는 기능이라 delete가 적당한 이름인것 같은데, delete로 짓기에는 중복인것 같아서요

Reply:
진짜 중복일지 아닌지 의도적 중복인지 여부는 해당 역할을 수행하는 객체의 책임과 역할을 가지고 이해하는게 좋습니다.

중복에 대한 내용이 친절하게 적혀있는 링크도 있군요

https://yrok.tistory.com/entry/%ED%81%B4%EB%A6%B0-%EC%95%84%ED%82%A4%ED%85%8D%EC%B2%98%EC%86%8C%ED%94%84%ED%8A%B8%EC%9B%A8%EC%96%B4-%EA%B5%AC%EC%A1%B0%EC%99%80-%EC%84%A4%EA%B3%84%EC%9D%98-%EC%9B%90%EC%B9%99Clean-Architecture-5%EB%B6%80-%EC%95%84%ED%82%A4%ED%85%8D%EC%B2%98-16%EC%9E%A5

Reply:
제 의견을 나열하자면 두 코드는 **결코** 중복이 아니라고 생각합니다.

RecruitmentService의 delete는 delete 행위를 수행하는 로직으로 생각하고 보면 되고,
RecruitmentController의 delete는 http delete 메소드를 핸들링하는 핸들러 목적으로 delete라는 네이밍을 지었으니, 서로 다른 관심사의 delete이기 때문이라고 보면 될것 같군요